src/cmdline.rs

```diff
@@ -136,6 +137,15 @@ pub fn parse_args() -> Result<Config> {
                         .help("Additional kernel args for the first boot")
                         .takes_value(true),
                 )
+                .arg(
+                    Arg::with_name("copy-network-config")
+                        .short("n")
+                        .long("copy-network-config")
+                        .value_name("path")
+                        .default_value("/etc/NetworkManager/system-connections/")
```

Comment:
This default value will likely need to be different for RHCOS (for now). I'm thinking we can just carry that as a patch in the RPM in downstream. Does that sound OK?

Reply:
We also do want to copy that path for RHCOS too.  So it's more like disabling ifcfg copying on FCOS.  We could do something like look at the *target* OS install and conditionalize based on that...since we do want to allow people to use the upstream/Fedora `coreos-installer` binary to install RHCOS.

Reply:
Or to rephrase, FCOS supports NM config files; RHCOS supports NM config files *and* ifcfg files.  It's not "RHCOS only supports ifcfg files".

Perhaps simplest is just to make this a boolean value, and copy files from both paths if they exist and the option is specified.

Reply:
> We also do want to copy that path for RHCOS too. So it's more like disabling ifcfg copying on FCOS. We could do something like look at the _target_ OS install and conditionalize based on that...since we do want to allow people to use the upstream/Fedora `coreos-installer` binary to install RHCOS.

Using the FCOS installer to install RHCOS is probably a little risky long term, though for this case the user could still pass `-n /etc/sysconfig/network-scripts/` and get it to work.

> Or to rephrase, FCOS supports NM config files; RHCOS supports NM config files _and_ ifcfg files. It's not "RHCOS only supports ifcfg files".
> 
> Perhaps simplest is just to make this a boolean value, and copy files from both paths if they exist and the option is specified.

Since the use case here is to propagate forward the files generated by `nmtui` I think we should support the default format that is spit out by `nmtui` on each platform and not overcomplicate things.

Reply:
> Since the use case here is to propagate forward the files generated by nmtui I think we should support the default format that is spit out by nmtui on each platform and not overcomplicate things.

Hmm.  I suppose you're right that NM will save in ifcfg files by default on RHCOS.  I wonder how hard that'd be to change though...i.e. make ifcfg files a *read* source but not a write target.

Reply:
> i.e. make ifcfg files a _read_ source but not a write target.

+1, that's what it does for nm keyfiles today IIUC. I'd have to dig into the plugin to see if it's possible. For now I'm going to stick with this and I'll come back and change it if I have enough time between now and this Friday.

Reply:
opened this against NM upstream to see if there's any way to do what we want: https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/415